combat-trainer.lic

```diff
@@ -4018,15 +4044,6 @@ class GameState
     weapon_training[skill]
   end
 
-  def necro_casting?
-    return false unless DRStats.necromancer?
-    return true if @prepare_cfb
-    return true if @casting_cfb
-    return true if @prepare_consume
-    return true if @casting_consume
-    false
-  end
```

Comment:
THIS METHOD HAS NOT BEEN REMOVED. 

It has been _moved_ further down to new line 4316 with some other magika methods.